bcda/auth/plugin/alpha.go

```diff
@@ -248,18 +247,44 @@ func revokeAccessTokenByID(tokenID uuid.UUID) error {
 	return db.Error
 }
 
-func (p *AlphaAuthPlugin) ValidateAccessToken(token string) error {
-	return errors.New("not yet implemented")
+func (p *AlphaAuthPlugin) ValidateAccessToken(tokenString string) error {
+	t, err := p.DecodeAccessToken(tokenString)
+	if err != nil {
+		return err
+	}
+
+	c := t.Claims.(*AllClaims)
+
```

Comment:
I think you should be checking the validity of the claim here  "t.Claims.(*AllClaims).Valid()" is checkabel based on internal locally stored info.  The other 2 checks rely on a DB call which is fundamentally more expensive than a local check.  If we can exit before doing them we should.

Reply:
✔️